spec/models/card_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Card, type: :model do
+  context 'when validations' do
+    let(:card) { create(:card) }
+
+    it 'when params presence' do
+      expect(card.valid?).to eq(true)
+    end
+    it 'when original_text presence' do
+      card.original_text = nil
+      expect(card.valid?).to eq(false)
+    end
+
+    it 'when translated_text presence' do
+      card.translated_text = nil
+      expect(card.valid?).to eq(false)
+    end
+
+    it 'when text not valid' do
+      card.translated_text = 'Дом'
+      card.original_text = 'дом'
+      expect(card.valid?).to eq(false)
```

Comment:
you also should check that card is invalid because you texts are not equal. Add message check pls.